cellprofiler/modules/identifyprimaryobjects.py

```diff
@@ -42,7 +45,9 @@
 -  The image should be grayscale.
 -  The foreground (i.e, regions of interest) are lighter than the
    background.
--  The image should be 2D, not volumetric.
+-  The image should be 2D. 2D slices of 3D images are acceptable if
+   loaded as such in the **Images** module. For volumetric analysis
```

Comment:
Can you change `2D slices of 3D images are acceptable if loaded as such in the **Images** module.` to `2D slices of 3D images are acceptable if the image has not been loaded as volumetric in the **NamesAndTypes** module.` ?

Reply:
done!